mobile/lib/repositories/database.repository.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+import 'package:immich_mobile/interfaces/database.interface.dart';
+import 'package:isar/isar.dart';
+
+const Symbol _zoneTxn = #zoneTxn;
```

Comment:
What is this for?

Reply:
I've added a comment to the code: copied from Isar; needed to check if an async transaction is already active